tools/bin/go_core_ccip_deployment_tests

```diff
@@ -3,26 +3,18 @@ set -o pipefail
 set +e
 
 SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
-OUTPUT_FILE="../output.txt"
-USE_TEE="${USE_TEE:-true}"
+OUTPUT_FILE=${OUTPUT_FILE:-"./output.txt"}
```

Comment:
This pathing has changed a bit, it used to be ‘../output.txt’ because this test cds into the deployment directory.

I think we need to keep it in the parent directory otherwise the artifact upload will not be able to find the output.

Reply:
good catch, @erikburt!